test/shared/box.lua

```diff
@@ -134,6 +134,28 @@ box.once('initialization', function()
     local tuple = yaml.decode(yml)[1]
     tuple[1] = "12345"
     box.space._schema:insert(tuple)
+
+    --gh-151 Support new _index system space format
+    -- After tarantool-1.7.5-153-g1651fc9be the new _index format
+    -- was introduced. When an index part uses is_nullable or
+    -- collation parameter, then the new format will be used.
+    if tarantool_version_at_least(1, 7, 6, 0) then
+        local test_new_format = box.schema.space.create('test_new_format', {
+            format = {
+                { type = compat.unsigned, name = 'f1'},
+                { type = compat.unsigned, name = 'f2', is_nullable = true},
+                { type = compat.string, name = 'f3'},
+            }
+        })
+
+        test_new_format:create_index('primary', {
+            parts  = {{1, compat.unsigned}}
+        })
+        test_new_format:create_index('secondary', {
+            parts  = {{2, compat.unsigned, is_nullable = true},
+            {3, compat.string, collation = 'unicode_ci'}}
```

Comment:
Nit: Strange indent. Maybe format is like so?

```
parts = {
    {<...>},
    {<...>},
}
```

Reply:
fixed